README.md

```diff
@@ -11,17 +11,16 @@
 </p>
 
 
-Vector is an [open-source][urls.vector_repo] utility for building observability
-pipelines. [Collect][docs.sources], [transform][docs.transforms], and
-[route][docs.sinks] log, metrics and events with one simple tool.
+Vector is a _highly reliable_ observability data router built for demanding
+production environments. Vector is designed on the following principles:
 
-Built in [Rust][urls.rust], Vector places high-value on
-[performance][pages.index#performance], [correctness][pages.index#correctness],
-and [operator friendliness][docs.administration]. It compiles to a single static
-binary and is designed to be [deployed][docs.deployment] across your entire
-infrastructure, serving both as a light-weight [agent][docs.roles.agent] and a
-highly efficient [service][docs.roles.service]. Take back ownership and control
-of your observability data with Vector.
+* **High reliability** - Built in [Rust][urls.rust], Vector is [memory safe][urls.rust_memory_safety], [correct][pages.index#correctness], and [performant][pages.index#performance].
+* **Operator safety** - Vector is pragmatic and hard to break. It avoids the common pitfalls in similar tools.
+* **All data** - [Logs][docs.data-model.log], [metrics][docs.data-model.metric], and traces (coming soon). A [sophisticated data model][docs.data-model] enables _correct_ interoperability.
+* **One tool** - Deploys as an [agent][docs.roles.agent] or [service][docs.roles.service]. One tool gets your data from A to B.
+
+Vector is **deployed over 100,000 times per day**, and is trusted by Fortune 500
+comapanies and forward thinking engineering teams.
```

Comment:
The "forward thinking" part kind of feels like this:

![](https://media.giphy.com/media/9Q249Qsl5cfLi/giphy.gif)

How about "... and engineering teams trying to tame vast networks of log pipelines." or something similar that invokes a feeling or some imagery.